src/libraries/System.Numerics.Tensors/src/System/Numerics/Tensors/Tensor.cs

```diff
@@ -691,6 +691,56 @@ private Tensor<T> GetTriangle(int offset, bool upper)
         /// <param name="value">The new value to set at the specified position in this Tensor.</param>
         public abstract void SetValue(int index, T value);
 
+        /// <summary>
+        /// The type that implements enumerators for <see cref="Tensor{T}"/> instances.
+        /// </summary>
+        public struct Enumerator : IEnumerator<T>
+        {
+            private readonly Tensor<T> tensor;
+            private int index;
```

Comment:
Nit: naming according the guidelines
```suggestion
            private readonly Tensor<T> _tensor;
            private int _index;
```

Reply:
Hmm, I believe the idea here is to follow the file's naming conventions:
https://github.com/dotnet/runtime/blob/aa327625efca011f8f96cbd59f002fd1b28d2f56/src/libraries/System.Numerics.Tensors/src/System/Numerics/Tensors/Tensor.cs#L258

Reply:
Yeah that was my thought process. I initially named with underscores but changed it to be consistent with the rest of the file.